resources/openapi/openapi.yaml

```diff
@@ -589,7 +589,7 @@ paths:
         content:
           '*/*':
             schema:
-              $ref: '#/components/schemas/QuerySpecDto'
+              $ref: '#/components/schemas/QuerySpecSchema'
```

Comment:
I think the `openapi.yaml` file isn't used for anything anymore, so we should be able to delete it.